### PR TITLE
Sggianel

### PR DESCRIPTION
**Pull request by samgianelli**
**Practice pull request**
Please make practice comments just to try out the system.

### CODE REVIEW

File: src/parallelLoop.java [hunk 1]
```diff
@@ -1,11 +1,11 @@
+import java.util.concurrent.*;
+import java.util.*;
 
 public class parallelLoop
```
Comment (samgianelli) [inline src/parallelLoop.java]: test comment

Comment (csenderling) [inline src/parallelLoop.java]: test reply

File: src/parallelLoop.java [hunk 2]
```diff
@@ -24,14 +24,14 @@ public static void main(String[] args)
 
         Runnable task = () -> {
             try {
-                /*Thread.sleep(5000);
+                Thread.sleep(5000);
```
Comment (csenderling) [inline src/parallelLoop.java]: test review

File: src/parallelLoop.java (hunk 1 above)
Comment (dsenderling1) [inline src/parallelLoop.java]: Testing as well

**Pull request merged**